packages/grafana-data/src/utils/location.test.ts

```diff
@@ -57,7 +57,7 @@ describe('locationUtil', () => {
     });
     test('absolute url with subdirectory subUrl', () => {
       const urlWithoutMaster = locationUtil.stripBaseFromUrl('http://www.domain.com:9877/thisShouldRemain/subUrl/');
-      expect(urlWithoutMaster).toBe('/thisShouldRemain/subUrl/');
+      expect(urlWithoutMaster).toBe('http://www.domain.com:9877/thisShouldRemain/subUrl/');
```

Comment:
this test must have been wrong? Why should  tripBaseFromUrl remove a domain to another web app on the same domain? in this test case the domain is `http://www.domain.com:9877` but grafana is under `/subUrl`  so this is a link to another web application on the same domain.

Reply:
I struggled with the reasoning of this too when working on the associated PR. It seems to have been added in relation to fixing a [datalinks bug]( https://github.com/grafana/grafana/pull/32634). 🤔

Reply:
well in that PR the test is correct  (http://www.domain.com:9877/subUrl/grafana/)

Reply:
Ah you're right, it's definitely wrong and I'm comparing two different tests (one with config.appSubUrl and one without). 🤦